util/src/main/java/io/kubernetes/client/util/Watch.java

```diff
@@ -120,4 +120,8 @@ public boolean hasNext() {
     public void remove() {
         throw new UnsupportedOperationException("remove");
     }
+    
+    public void close() throws IOException {
```

Comment:
I'm wondering if this is meant to go in `remove()` - @brendandburns?

Reply:
No, `remove()` is part of implementing the Java `Iterable` interface, (and it's legal for it to throw if you don't support it...)

We should mark `remove()` with `@Override` to make this clear.